packages/tools/webpack-fluid-loader/README.md

```diff
@@ -41,10 +42,13 @@ provided in the following ways (looked for in the following order):
 
 ### command line:
 ```
+npm run start -- --env.discoveryEndpoint https://discoveryEndpoint.com --env.tenantId my_tenant --env.tenantSecret my_secret --env.bearerSecret bear_secret --env.npm npm.com
```

Comment:
Since we're adding this, #9615 updated Webpack CLI to v4 which changes how these need to be passed. Can you update both of these to use the new syntax?
```
npm run start -- --env discoveryEndpoint=https://discoveryendpoint.com --env tenantId=my_tenant --env tenantSecret=my_secret --env bearerSecret=bear_secret --env npm=npm.com
```
and
```
npm run start -- --env fluidHost=https://fluidhost.com --env tenantId=my_tenant --env tenantSecret=my_secret --env bearerSecret=bear_secret --env npm=npm.com
```

Reply:
Got it. Thanks. Done. Please have a check... :)